auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -1137,7 +1149,7 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   justify-content: space-between;
 
   h1 {
-    margin-bottom: -10px;
+    margin-bottom: -30px;
```

Comment:
Why not simply remove (or adjust) the bottom margin on "view-header"?

Reply:
I will see if I can reduce this space with some more CSS adjustment. After removing the bottom margin:
![image](https://github.com/bcgov/sbc-auth/assets/116035339/76b53b58-9895-4bc1-9500-6f1911c1ccf2)

Reply:
If I delete the margin-bottom on the H1, and change the margin-bottom to 0.75rem on the view-header, it looks like this for me:

![image](https://github.com/bcgov/sbc-auth/assets/10002889/34b12ff6-99ea-488c-b7f8-846549e8c292)

Reply:
Why do your screenshots keep showing "Add an existing Business or Name Request" ? That's main branch code, not feature branch code. (But your PR and temp URL look correct.)

Reply:
Yes that is exactly what I found. Override .view-header margin.

Reply:
> > Why do your screenshots keep showing "Add an existing Business or Name Request" ? That's main branch code, not feature branch code. (But your PR and temp URL look correct.)
> 
> I am pretty sure I branched off from feature branch. I will double check why it shows main branch code

I think this is from another PR. Not sure but looks like this PR https://github.com/bcgov/sbc-auth/pull/2469

Reply:
Oh well, this PR's changes, and the temp URL, look ok.

Reply:
Yes. It is FF so in DEV it looks different than the temp URL.

Reply:
So you were taking screenshots from Dev, not from the temp URL or from your local? That would explain it.